nf_core/components/components_test.py

```diff
@@ -38,7 +41,7 @@ def __init__(
         self.no_prompts = no_prompts
         self.pytest_args = pytest_args
 
-        super().__init__(".", remote_url, branch, no_pull)
+        super().__init__("component_type", ".", remote_url, branch, no_pull)
```

Comment:
```suggestion
        super().__init__(component_type, ".", remote_url, branch, no_pull)
```

Reply:
I think this was a placeholder because `component_type` also needs to be added to the `__init__` for `ComponentTest`?

Reply:
We don't have a `component_type `in `ComponentsTest`, that's why I used a string to fill the variable. But maybe I am missing something and it's better to add it.